app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadFragment.kt

```diff
@@ -44,13 +45,20 @@ class SellPayloadFragment :
             widgetSendDetails.onInputFocusChanged = View.OnFocusChangeListener { _, isFocused ->
                 if (isFocused) sellAnalytics.logSellTokenAmountFocused()
             }
-            widgetSendDetails.focusInputAndShowKeyboard()
 
             buttonCashOut.setOnClickListener {
                 sellAnalytics.logSellSubmitClicked()
                 presenter.cashOut()
             }
         }
+        childFragmentManager.setFragmentResultListener(
+            SellInformationBottomSheet.SELL_INFORMATION_REQUEST_KEY,
+            viewLifecycleOwner
+        ) { _, bundle ->
+            if (bundle.getBoolean(SellInformationBottomSheet.SELL_INFORMATION_RESULT_KEY)) {
```

Comment:
make static imports please

Reply:
ok, but then why we use [this rule](https://github.com/p2p-org/key-app-android/wiki/Android-Code-Style#221-fields-definition-and-naming) ?

Reply:
How this rule is related to this place?